app/javascript/app/components/ndc-sdg/ndc-sdg-linkages-content/ndc-sdg-linkages-content-component.jsx

```diff
@@ -17,20 +18,31 @@ class NdcSdgLinkagesContent extends PureComponent {
       goalHover,
       targetHover,
       handleGoalHover,
+      handleGoalTap,
       handleTargetHover
     } = this.props;
     return (
-      <div className={cx(layout.content, styles.grid)}>
-        <NdcsSdgsMetaProvider />
-        <NdcSdgLinkagesTable
-          goalHover={goalHover}
-          onGoalHover={handleGoalHover}
-          targetHover={targetHover}
-          onTargetHover={handleTargetHover}
-        />
-        <NdcSdgLinkagesMap goalHover={goalHover} targetHover={targetHover} />
-        <ModalMetadata />
-      </div>
+      <TabletLandscape>
+        {matches => (
+          <div
+            className={cx({ [layout.content]: matches === true }, styles.grid)}
+          >
+            <NdcsSdgsMetaProvider />
+            <NdcSdgLinkagesTable
+              goalHover={goalHover}
+              onGoalHover={handleGoalHover}
+              onGoalTap={handleGoalTap}
+              targetHover={targetHover}
```

Comment:
And I'm not sure we need this anymore, could you review it?

Reply:
Maybe the name of the Tap event is a little bit misleading, because what we are really handling is the click event, but we are triggering this handler on goal click to set the proper data in the map on mobile devices (thats why I call it onTap, to understand that this handler aims touch devices).